### DISCUSSION TITLE
how to setting custom saved directory?

### DISCUSSION THREAD

**Discussion by dablro12** [tcgoetz/GarminDB#224] [Q&A] [answered]
thank your project,

when i started command "garmindb_cli.py --all --download --import --analyze", my healthy-data is saved to local directory.
How to select saved directory for my DB?

**Comment by tcgoetz** [comment, marked as answer]
Currently you have to edit config.py component of the package and change "base_dir" element.

**Reply by dablro12** [reply]
thank your rapdily comment. I try it!

**Comment by tcgoetz** [comment]
Changes on the development branch allow setting the data directory in the JSON config file. Those changes will be in the next release.

**Comment by dablro12** [comment]
Thank you, your reply!
I hope to release setting saved directory!
 so I wait for your next release!

2024년 3월 11일 (월) 08:41, Tom Goetz ***@***.***>님이 작성:

> Changes on the development branch allow setting the data directory in the
> JSON config file. Those changes will be in the next release.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tcgoetz/GarminDB/discussions/224#discussioncomment-8738844>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AM7L2LBAG62OKREV3XRNVRLYXT4RZAVCNFSM6AAAAABEJGLJ26VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DOMZYHA2DI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 



[image: image.jpeg]

최대현   바이오메디컬공학 및 AI&Laguage
m.

010-5661-6371
e.

***@***.***
p.

choedaehyeon Portfolio <https://valiant-barnacle-6bf.notion.site/>
